apps/src/templates/progress/ProgressBubble.jsx

```diff
@@ -230,6 +247,23 @@ class ProgressBubble extends React.Component {
                 </span>
               )}
             </div>
+            {experiments.isEnabled(experiments.MINI_RUBRIC_2019) &&
+              levelIsAssessment &&
+              !smallBubble &&
+              !dontShowAssessmentIcon && (
```

Comment:
(i know you just changed the name of this prop, so i'm sorry in advance!)

this would be clearer as `showAssessmentIcon` because it would remove the double-negative here (`!dontShowAssessmentIcon == "don't not show assessment icon"`)

Reply:
I don't know that it makes sense to name it showAssessmentIcon since we only set the prop when we know we don't want to show the assessment icon. There are cases when that prop would not be set and we would want to show the assessment icon.

Reply:
ohh, i see. maybe `hideAssessmentIcon` would be more explicit if that's the special case we're solving with this prop

Reply:
I could make it something like hideAssessmentIcon so its not a double negative?

Reply:
haha we had the same suggestion at the same time! that sounds good 😄